File: 8kyu/(8 kyu) Sum of positive/(8 kyu) Sum of positive.cs
using System;
using System.Linq;

public class Kata
{
  public static int PositiveSum(int[] arr)
  {
    int res = 0;
    foreach (int x in arr)
    {
      if (x > 0) res += x;
    }
    return res;
  }
}

File: 8kyu/(8 kyu) Sum of positive/(8 kyu) Sum of positive.java
public class Positive{
  public static int sum(int[] arr){
    int res = 0;

    for (int i = 0; i < arr.length; i++) {
        if (arr[i] > 0) {
            res += arr[i];
        }
    }

    return res;
  }
}
